Message.java

```diff
@@ -0,0 +1,20 @@
+
+public class Message {
+	
+	private String toAddress;
+	private String fromAddress;
+	private String message;
+	
+	public void getTo(String toAddress) {
```

Comment:
Traditionally, get/set methods are used to either 

- return a value contained in a private member

- set the value within a private member

It looks like you're trying to use a get method to set a member value, when _instead_ you should use a _set_ method to assign a value to private member.

Use get methods to return the value that is contained within the private member